src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -97,5 +97,17 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
 
   </div>
 
+  <div id="container">
+
+    <h1><Members>
+      <a href="" style="float: right">&#8635;</a></h1>
+    <div id="members">
+      <ul>
+    	<li>Members: <%=request.getAttribute("getMembers")%> </li>
```

Comment:
Have you run this locally? I would be surprised if this works given that we are not setting any attribute "getMembers" on the request. We would have to set this in the ChatServlet the same way we set the attribute for "conversation" and "messages" : https://github.com/summerwinter7/CodeU-Summer-2018/blob/f14c29b1ed1a8d93ada3d1f324b7f9ea9c4b54f9/src/main/java/codeu/controller/ChatServlet.java#L103

Here we would call the method that Summer adds in her PR, conversation.getMembers() to get a list of users to display.

Reply:
It just says null when I run it because there are no members in the group convo. I did the same thing with Admin and that works fine, but I can change it if it doesn't work after the updates.

Reply:
I just merged my code, so I think now in the ChatServlet file you can do request.setAttribute("members") and pass in the list of members from that conversation using conversationl.getMessages(). The list is storing UUIDs now instead of actual Users, so you might need to convert them in the ChatServlet as well.